app-emulation/punes/punes-0.105.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools xdg-utils
+
+DESCRIPTION="Nintendo Entertainment System (NES) emulator"
+HOMEPAGE="https://github.com/punesemu/puNES"
+SRC_URI="https://github.com/punesemu/puNES/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
This license isn't correct ;)

Reply:
License is GPL-2: https://github.com/punesemu/puNES/blob/master/COPYING

Reply:
Check the .c files. 
https://devmanual.gentoo.org/general-concepts/licenses/index.html and read this.